### PR TITLE
testing: add Kokoro builds

### PR DESCRIPTION
**Pull request by tmatsuo**
part of #6

### CODE REVIEW

File: tests/python/test_envvar.py [hunk 1]
```diff
@@ -20,21 +20,36 @@ def test_envvar():
     assert 'TRAMPOLINE_VERSION' in os.environ
     assert os.environ['TRAMPOLINE_VERSION'][0] == '2'
     assert 'RUNNING_IN_CI' in os.environ
+    # This is set in .travis.yml or .kokoro/python/common.cfg
+    assert os.environ.get('TEST_ENV') == 'test'
     if os.environ['RUNNING_IN_CI'] == 'true':
+        ci_envvars_tested = false
```
Comment (chingor13) [inline tests/python/test_envvar.py]:
```suggestion
        ci_envvars_tested = False
```

**Review by chingor13 — CHANGES_REQUESTED** [review]

File: tests/python/test_envvar.py (hunk 1 above)
Comment (tmatsuo) [reply tests/python/test_envvar.py]: Yup, thanks

**Comment by tmatsuo** [comment]
It correctly failed!

**Comment by tmatsuo** [comment]
@chingor13 Now the tests are passing. PTAL

**Review by chingor13 — APPROVED** [review]

**Pull request merged**